docs/contracts/v4/guides/11-position-manager.mdx

```diff
@@ -0,0 +1,838 @@
+    ignoreDust ? 3 : 2
+);
+
+// Parameters for DECREASE_LIQUIDITY
+params[0] = abi.encode(
+    tokenId,           // Position to decrease
+    liquidityDecrease, // Amount to remove
+    amount0Min,        // Minimum token0 to receive
+    amount1Min,        // Minimum token1 to receive
+    ""                // No hook data needed
+);
+```
+
+Set up delta resolution parameters:
+
+```solidity
+if (ignoreDust) {
+    // Using CLEAR_OR_TAKE with dust thresholds
+    uint256 dustThreshold = 1000; // Example threshold
+    params[1] = abi.encode(currency0, dustThreshold);
+    params[2] = abi.encode(currency1, dustThreshold);
+} else {
+    // Standard TAKE_PAIR for receiving tokens
+    params[1] = abi.encode(
+        currency0,
+        currency1,
+        recipient
+    );
+}
+```
+
+Execute the operation:
+
+```solidity
+// Execute the decrease
+    positionManager.modifyLiquidities(
+        abi.encode(actions, params),
+        block.timestamp + 60  // 60 second deadline
+    );
+}
+```
+
+## Collecting Fees
+
+In v4’s Position Manager, there isn’t a dedicated COLLECT command. Instead, fees are collected by using DECREASE_LIQUIDITY with zero liquidity. This pattern leverages the fact that fees are automatically credited during liquidity operations.
+
+### **Understanding Fee Collection**
+
+When collecting fees, you need to:
+
+1. Perform a DECREASE_LIQUIDITY operation with zero liquidity
+2. Handle the positive deltas (the fees you’re collecting)
+3. Specify where the fees should go
+
+### **Implementation**
+
+```solidity
+/// @notice Collects accumulated fees from a position
+/// @param tokenId The ID of the position to collect fees from
+/// @param recipient Address that will receive the fees
+function collectFees(
+    uint256 tokenId,
+    address recipient
+) external {
+
+```
+
+Set up our operation sequence based on dust handling preference:
+
+```solidity
+// Define sequence of operations:
+// 1. DECREASE_LIQUIDITY with zero amount to collect fees
+// 2. Either TAKE_PAIR or CLEAR_OR_TAKE for receiving tokens
+bytes memory actions;
+if (ignoreDust) {
+    actions = abi.encodePacked(
+        Actions.DECREASE_LIQUIDITY,  // Zero liquidity decrease
+        Actions.CLEAR_OR_TAKE,       // Handle token0 fees
+        Actions.CLEAR_OR_TAKE        // Handle token1 fees
+    );
+} else {
+    actions = abi.encodePacked(
+        Actions.DECREASE_LIQUIDITY,  // Zero liquidity decrease
+        Actions.TAKE_PAIR           // Take all fees
+    );
+}
+```
+
+Prepare the parameters array:
+
+```solidity
+// Set up parameters array
+bytes[] memory params = new bytes[](
+    ignoreDust ? 3 : 2
+);
+
+// Parameters for DECREASE_LIQUIDITY
+// All zeros since we're only collecting fees
+params[0] = abi.encode(
+    tokenId,  // Position to collect from
+    0,        // No liquidity change
+    0,        // No minimum for token0 (fees can't be manipulated)
+    0,        // No minimum for token1
+    ""       // No hook data needed
+);
+```
```

Comment:
lets update and remove the dust logic -- when youre collecting fees you probably dont want to forfeit dust

Reply:
all dust logic removed. Please make sure I didn't mistakenly remove any important logic.